src/app/components/footer/footer.html

```diff
@@ -1,38 +1,41 @@
-<footer class="p-60" style="padding-bottom: 10px">
+<footer class="p-60" style="background-color:#262626; padding-bottom: 10px">
```

Comment:
Did you do this? Or this something clashing with older commits?

Reply:
Comes from older commits, sorry